docs/users.rst

```diff
@@ -895,12 +895,14 @@ build.
 Override Parent Image
 ----------------------
 
-The parent image used for building a layered image is determined by the ``FROM``
-instruction in the Dockerfile by default. Users can override this behavior
-by specifying a koji parent build via the ``koji_parent_build`` API parameter.
-When given, the image reference in the provided koji parent build will be used as
-the value of the FROM instruction. The same source registry restrictions
-apply.
+OSBS uses the ``FROM`` instruction in the Dockerfile to find a parent image
+for a layered image build. Users can override this behavior by specifying a
+koji parent build via the ``koji_parent_build`` API parameter in the
+``buildContainer`` task. When a user specifies a ``koji_parent_build``
```

Comment:
The `koji_parent_build` can also be passed through the osbs cli or api (wihtout going through a koji  task or the koji-containerbuild plugin). This is not the most straightforward way of triggering a build, but could still be used. Since this is the upstream documentation,could we rephrase this bit to make that clear?

Reply:
Oh, yeah, I should not have added the `buildContainer task` phrase. I've deleted it and re-pushed this branch.